tests/getMovieScreenings.test.js

```diff
@@ -0,0 +1,105 @@
+import { getMovieScreenings } from "../src/movies";
+
+describe('getMovieScreenings', () => {
+
+    test('No old screenings should appear', async () => {
+
+        const screenings = await getMovieScreenings({
+            loadAllMovieScreenings: async () => ({
+                    "data": [
+                        {
+                          "id": 203,
+                          "attributes": {
+                            "start_time": "2023-06-15T21:00:00.000Z",
+                            "room": "Stora salongen",
+                            "createdAt": "2023-05-25T13:09:54.278Z",
+                            "updatedAt": "2023-05-25T13:09:54.278Z"
+                          }
+                        },
+                        {
+                          "id": 204,
+                          "attributes": {
+                            "start_time": "2024-03-30T12:00:00.000Z",
+                            "room": "Stora salongen",
+                            "createdAt": "2024-01-29T19:08:41.554Z",
+                            "updatedAt": "2024-01-29T19:08:41.554Z"
+                          }
+                        },
+                    ]               
+                })
+        }, 3)
+        expect(screenings.length).toEqual(1)
+    })
+
+    test('Dates should be in order', async () => {
+        const screenings = await getMovieScreenings({
+            loadAllMovieScreenings: async () => ({
+                    "data": [
```

Comment:
There is a big difference in indentation here. Typically, an object like this would be indented only one level (4 spaces). I recommend using automatic formatting in your editor to not have to worry about these things.